gravitee-apim-console-webui/src/management/api/api-navigation/api-federated-menu.service.spec.ts

```diff
@@ -109,12 +109,12 @@ describe('ApiFederatedMenuService', () => {
             routerLink: '',
             tabs: [
               {
-                displayName: 'Default Pages',
-                routerLink: 'v4/documentation/default-pages',
+                displayName: 'Main Pages',
```

Comment:
we can have multiple Main Pages ? it's not just for Home?

Reply:
From what I understood when we talked about it, the idea is to add in this "section" the General condition pages you can attach to plans

Reply:
Yes, the idea is to have a "Terms and Conditions" section in the future under Main Pages